xrpl/wallet/main.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         """
 
         pk, sk = derive_keypair(self.seed, algorithm=algorithm)
+        self.algorithm = algorithm
```

Comment:
Is this just a random bugfix? (Or is this a new feature?)

Reply:
Whoops, left a debugging statement in